frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -837,6 +850,97 @@ public boolean isSnapshotDisabled() {
         return snapshotDisabled;
     }
 
+    // Imports the token class and sets the expiration time either default or custom
+    // calls generate key file in token api to create 3 keys and logs the result
+    public void setupTokenClass() {
+        try {
+            tokenClass = Class.forName("org.pytorch.serve.plugins.endpoint.Token");
+            tokenObject = tokenClass.getDeclaredConstructor().newInstance();
+            Method method = tokenClass.getMethod("setTime", Integer.class);
+            if (prop.getProperty(TS_TOKEN_EXPIRATION_TIME) != null) {
+                timeToExpiration = Integer.valueOf(prop.getProperty(TS_TOKEN_EXPIRATION_TIME));
+            }
+            method.invoke(tokenObject, timeToExpiration);
+            method = tokenClass.getMethod("generateKeyFile", Integer.class);
+            if ((boolean) method.invoke(tokenObject, 0)) {
+                System.out.println("TOKEN CLASS IMPORTED SUCCESSFULLY");
+                dumpKeyLogs();
+            }
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
+        } catch (NoSuchMethodException
+                | IllegalAccessException
+                | InstantiationException
+                | InvocationTargetException e) {
+            e.printStackTrace();
+        }
+        tokenAuthorizationEnabled = true;
```

Comment:
please add log to explicitly notify user that token is not enabled successfully.

Reply:
Noted